src/diffusers/modular_pipelines/modular_pipeline.py

```diff
@@ -78,139 +76,59 @@ class PipelineState:
     [`PipelineState`] stores the state of a pipeline. It is used to pass data between pipeline blocks.
     """
 
-    inputs: Dict[str, Any] = field(default_factory=dict)
-    intermediates: Dict[str, Any] = field(default_factory=dict)
-    input_kwargs: Dict[str, List[str]] = field(default_factory=dict)
-    intermediate_kwargs: Dict[str, List[str]] = field(default_factory=dict)
+    values: Dict[str, Any] = field(default_factory=dict)
+    kwargs_mapping: Dict[str, List[str]] = field(default_factory=dict)
```

Comment:
Since we've removed the distinction between inputs and intermediates, we can perhaps simplify `PipelineState` as well. 

All values are stored under `value` and all `kwargs_type` under `kwargs_mapping`. Setting and getting are through `set`/`get` methods which handle both single string/list inputs for setting/fetching.